x/ibc/07-tendermint/types/client_state.go

```diff
@@ -0,0 +1,47 @@
+package types
+
+import (
+	clientexported "github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+)
+
+var _ clientexported.ClientState = ClientState{}
+
+// ClientState from Tendermint tracks the current validator set, latest height,
+// and a possible frozen height.
+type ClientState struct {
```

Comment:
Where are we storing the headers/consensus-state? Aren't they associated with the client state?

Reply:
yes, we are going to store them per-prefix too